src/styles/blocks/page.scss

```diff
@@ -0,0 +1,6 @@
+.page {
+  font-family: $PrimaryFont;
```

Comment:
The variable `$PrimaryFont` should be defined in the `_variables.scss` file as per the code style rules.